docs/src/main/asciidoc/chapters/administration.txt

```diff
@@ -1182,17 +1182,19 @@ conditions that vary from the environments individually tested in the various
 components. For example, Accumulo's use of HDFS includes many short block
 reads, which differs from the more common full file read used in most
 map/reduce applications. We have found that certain versions of Accumulo and
-Hadoop will include stability bugs that greatly affect overall stability. In
-our testing, Accumulo 1.6.2, Hadoop 2.6.0, and Zookeeper 3.4.6 resulted in a
-stable VM clusters that did not fail a month of testing, while Accumulo 1.6.1,
-Hadoop 2.5.1, and Zookeeper 3.4.5 had a mean time between failure of less than
-a week under heavy ingest and query load. We expect that results will vary with
-other configurations, and you should choose your software versions with that in
-mind.
-
-
-
-
-
-
-
+Hadoop will include stability bugs that greatly affect overall stability.
+Release notes typically contain information about versions used in
+release testing.
+
+The following table shows the Hadoop, Zookeeper and
+Thrift versions defined in the dependency section of the POM for building the
+artifacts.  
+
+|================================================
+|Accumulo |Hadoop |Zookeeper |Thrift
+|1.7            |2.2.0     |3.4.6          |0.9.1
+|1.8            |2.6.4     |3.4.6          |0.9.3
+|================================================
+
```

Comment:
> We don't need to update on every Hadoop bug

Wasn't suggesting that.

Take a hypothetical to describe what I was asking: We find that the `AccumuloInputFormat` is broken on Hadoop 2.8-present due to some runtime error which we did not realize (some subtle implementation which was not clear from the API and we did the wrong thing). We make fix to `AIF` in 1.7.4, 1.8.1, 1.9.0. Do we update all 1.7.0, 1.7.1, 1.7.2, 1.8.0 release notes to make users aware that these versions of Accumulo do not work with Hadoop 2.8-present? Or, do the release notes only get updated for 1.7.4, 1.8.1, 1.9.0 saying that "the previous versions were broken and are not any more"?